src/components/PresetControls/PresetControls.tsx

```diff
@@ -0,0 +1,74 @@
+import { useRef } from 'react';
+import { LevaButton } from '../LevaButton/LevaButton';
+import { exportPreset, importPreset } from '../../utils/presetUtils';
+import FileDownloadOutlinedIcon from '@mui/icons-material/FileDownloadOutlined';
+import FileUploadOutlinedIcon from '@mui/icons-material/FileUploadOutlined';
+import styles from './PresetControls.module.scss';
+
+export interface PresetControlsProps {
+  controls: Record<string, any>;
```

Comment:
I think `any` type can be reduced by:

```ts
import { type useLevaControls } from '../../Controls';

export interface PresetControlsProps {
  controls: ReturnType<typeof useLevaControls>['controls'];
  controlsAPI: ReturnType<typeof useLevaControls>['controlsAPI'];
  lang: ReturnType<typeof useLevaControls>['lang'];
}
```

or you can export that `return type` inside of  `Controls.tsx` ?